src/UsersModule/Components/LandingPage/LandingPage.tsx

```diff
@@ -1,17 +1,941 @@
+          </Grid>
+        </Box>
+
+        {/* Ads */}
+
+        <Box>
+          <SectionLabel label="Most popular ads" />
+          {isLoading ? (
+            <Loading />
+          ) : ads?.length > 0 ? (
+            <Grid
+              container
+              component="main"
+              sx={{
+                height: "65vh",
+                justifyContent: "center",
+                overflow: "auto",
+              }}
+              // className={`pageOverflow`}
+            >
+              <Grid
+                item
+                xs={10}
+                sm={6}
+                md={4}
+                lg={3}
+                sx={{
+                  height: "95%",
+                  // width: "100%",
+                  paddingY: 3,
+                }}
+                className={` ${styles.imgContainer}`}
+                position={'relative'}
+              >
+                <PriceContainer price={fiveAds[0]?.room?.price}/>
+                <Container
+                  sx={{
+                    height: "100%",
+                    width: "90%",
+                    backgroundImage: `url(${fiveAds[0]?.room?.images[0]})`,
+                    backgroundRepeat: "no-repeat",
+                    borderRadius: "20px",
+                    bgcolor: "black",
+                    backgroundSize: "cover",
+                    backgroundPosition: "center",
+                    // position:'relative'
+                    display: "flex",
+                    alignItems: "end",
+                    justifyContent: "start",
+                  }}
+                >
+                  <Box
+                  // position={'absolute'} bottom={0} left={0}
+                  >
+                    <CardData
+                      titleColor="white"
+                      descreptionColor="white"
+                      title={adsData[0].title}
+                      description={adsData[0].description}
+                    />
+                    
+                  </Box>
+                </Container>
+                <Container
+                  sx={{
+                    borderRadius: "20px",
+                    width: "90%",
+                    height: "100%",
+                    position: "relative",
+                    bottom: "100%",
+                    bgcolor: "#203FC736",
+                    display: "none",
+                    alignItems: "center",
+                    justifyContent: "center",
+                  }}
+                  className={`${styles.imgLayout}`}
+                >
+                  <button
+                    className="bg-transparent border-0 "
+                    onClick={() => addtofav(fiveAds[0]?.room?._id)}
+                  >
+                    <FavoriteOutlined sx={{ color: "white" }} />
+                  </button>
+                  <button
+                    className="bg-transparent border-0"
+                    onClick={() => goToRoomDetails(fiveAds[0]?.room?._id)}
```

Comment:
You'll notice that the id is added to the URL with the quotation mark, we need to fix this.